### PR TITLE
Carol/docs/libraries ca

### PR DESCRIPTION
**Pull request by catong**
This PR addresses a review comment from @kelketek on https://github.com/edx/edx-documentation/pull/102/
Please review @kelketek @lamagnifica 
FYI @antoviaque @mhoeber @srpearce @marcotuts

### CODE REVIEW

**Comment by catong** [comment]
@lamagnifica Quick sanity check by you please, if you have a minute?

File: en_us/course_authors/source/building_course/creating_new_course.rst [hunk 1]
```diff
@@ -73,7 +73,15 @@ Create a New Course
 
 4. Click **Save.**
 
-You then see the empty Course Outline.
+The page refreshes to show the empty **Outline** for your new course. The course
+identifier appears in the browser address bar as the final part of the URL in
+the format ``{key type}:{org}+{course}+{run}``, for example
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/creating_new_course.rst]: change comma to period and start new sentence?

File: en_us/course_authors/source/building_course/creating_new_course.rst [hunk 2]
```diff
@@ -73,7 +73,15 @@ Create a New Course
...
+``course-v1:edX+DemoX+Demo_2014``.
+
+  .. note::  In October 2014, identifiers for some new courses began to use the
```
Comment (lamagnifica) [inline en_us/course_authors/source/building_course/creating_new_course.rst]: That was true in October, but all new courses now use the new format, not just some of them... maybe instead,

.. note:: This URL format applies to all courses created after January 14, 2015. Older courses can have URLs  in the format...

File: en_us/course_authors/source/creating_content/libraries.rst [hunk 3]
```diff
@@ -19,7 +19,9 @@ content libraries.
 
 .. note:: Content libraries are available only for courses whose course
```
Comment (lamagnifica) [inline en_us/course_authors/source/creating_content/libraries.rst]: whose --> that have

**Comment by lamagnifica** [comment]
some suggestions inline

**Comment by catong** [comment]
Merging https://github.com/edx/edx-documentation/pull/112

**Pull request merged**